migration_toolkit_for_containers/installing-mtc.adoc

```diff
@@ -6,18 +6,18 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can install the {mtc-full} ({mtc-short}) on {product-title} 4.
+You can install the {mtc-full} ({mtc-short}) on {product-title} 4 in a networked environment.
```

Comment:
Can you write more about the networked environment?  Is there a connectivity requirement?

If this is a conventionally installed cluster that is not in a restricted network, then I believe conventionally installed clusters do not receive any further clarification.

Reply:
I could remove that. Just thought it seemed a little too terse.